code/game/machinery/computer/orders/order_computer/mining_order.dm

```diff
@@ -5,11 +5,13 @@
 	icon_state = "mining"
 	icon_keyboard = null
 	icon_screen = null
+	cargo_cheaper = TRUE
 	circuit = /obj/item/circuitboard/computer/order_console/mining
 
 	cooldown_time = 10 SECONDS //just time to let you know your order went through.
-	express_cost_multiplier = 1.5
-	uses_ltsrbt = TRUE
+	express_cost_multiplier = 1
+	var/cargo_shipping_multiplier = 0.65
```

Comment:
This is absolutely not how it should be done.

The cost shown on the mining order console should be the price of using the cargo shuttle as that is the "default" way of purchasing things, not express. Golem vendor has no cost difference so it is their default too.

You need to instead change the costs of the items themselves.

Reply:
i see no reason to do this

Reply:
"Express" is very fast so I do think it should be called that
Also the slow option is cheaper than express

Reply:
The base price should be the cost of using the cargo shuttle, the price is increased through express. The "default" (so the final cost shown in the UI) should be the price of using the cargo shuttle, because that's the more important number players should be worried about.

Reply:
the whole point here is to make miners use the cargo shuttle through positive reinforcement (cheap gear)
and still be able to get their items instantly for the normal (not cheap) price for cases such as megafauna about to obliterate mining and the miner needs a luxury pen to combat it

Reply:
They will still have cheap gear by using the cargo shuttle, the point is that the cargo shuttle is standard, express is optional.

Reply:
Express be standard, cargor optional

Reply:
hell even the tooltips change so people can figure out which way works best for them

Reply:
The intent of the vendor was that the cargo shuttle would be the standard, I wrote about this in the 'Potential Changes' section of the hackmd that made this change: https://hackmd.io/ImTe5FLeTgmI7spTWKBaFQ?view#Potential-Changes

Reply:
why should I refer to that when the original changes being shit is what sprouted this PR?

Reply:
hell some miners can make cargo purchases their "standard" and others can make express their standard

Reply:
Miners can do that, however the game (through its UI) should treat the cargo shuttle as the standard.

Reply:
> however the game (through its UI) should treat the cargo shuttle as the standard.
it has bloody tooltips

Reply:
![image](https://user-images.githubusercontent.com/70376633/210133042-81a628de-4e23-4676-9e3b-4f18327e24af.png)
![image](https://user-images.githubusercontent.com/70376633/210133052-e36f8741-9a6e-4ad6-ad7d-23fef98ad865.png)

Reply:
And there's no indication of what the actual price is when going through cargo, the only price shown is when using express. It should be the other way around, price increases on express, what is shown is cargo.

Reply:
that is entirely against the point of the PR

Reply:
im sure any miner knows basic maths

Reply:
changed purchase to say 35% cheaper and express to say at normal price

Reply:
If you're not willing to change it then I'm not willing to merge the PR, the hackmd says potential future changes and I'm telling you to go with what the hackMD says instead of lowering the prices of the items you want instead.

If you make the golem vendor cost the pre-changed costs to still make cargo cheaper than them to do your point of 'not make ghost roles targets', then you can do that if you want, but I do not want the idea of making express the default, it's called express for a reason, being that it's the optional "faster" choice.

Reply:
bro you high as fuck

Reply:
> If you make the golem vendor cost the pre-changed costs to still make cargo cheaper than them to do your point of 'not make ghost roles targets', then you can do that if you want, but I do not want the idea of making express the default, it's called express for a reason, being that it's the optional "faster" choice.

golem gear costs the normal price WHICH IS HOW IT SHOULDVE FUCKING BEEN (also cant do cargo deliveries)

anyhow please ping me on the tg discord my tag is jimmyl because this is seriously too slow to argue on

Reply:
![image](https://user-images.githubusercontent.com/70376633/210134416-8fa7e862-5db8-4af6-a9d0-5ea40317788a.png)
look at your fucking like to dislike ratio

Reply:
also like i have failed to understand the vast majority of your comments and im concluding that you want me to rename the buttons? is that correct?